components/61-layouts/filter-layout/_filter-layout.scss

```
@@ -12,6 +12,10 @@
       width: span(3 of 12);
       padding-right: gutter();
     }
+
+    &.modal {
+      max-height: unset;
```

`unset` is not supported for IE.... https://developer.mozilla.org/en-US/docs/Web/CSS/unset

```suggestion
      max-height: none;
```